README.md

```diff
@@ -32,7 +31,7 @@ by opening the inspector (navigate to http://localhost:21000/inspector in your w
 want to manually launch another instance of the worker, run the following command:
 
 ```
-cargo run --example project-example -- --worker-id RustWorker999 --worker-type RustWorker receptionist
+cargo run -- --worker-id RustWorker999 --worker-type RustWorker receptionist
```

Comment:
Does this command actually work in the root directory?

Reply:
No, it need to be run in the `project-example` directory. I'll update the wording to clarify :+1: